test/clients/interface.go

```diff
@@ -0,0 +1,23 @@
+package clients
+
+import (
+	"context"
+)
+
+type TestError struct {
+	Bucket string
```

Comment:
@jim-minter can you give me more details on how you envisioned the usage of `TestError.Bucket`?